core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -68,73 +68,75 @@ protected function displayException(Exception $exception)
             ? $this->developerError($exception, $msg, $code, $trace_details)
             : $this->genericError($msg, $code);
         // start gathering output
-        $output = '
-<div id="ee-error-message" class="error">
-    ' . $error_message . '
-</div>';
-        $styles = $this->exceptionStyles();
+        $output = '<div id="ee-error-message" class="error">
+            ' . $error_message . '
+        </div>';
         $scripts = $this->printScripts(true);
         if (defined('DOING_AJAX')) {
-            echo wp_json_encode(array('error' => $styles . $output . $scripts));
+            echo wp_json_encode(array('error' => $output . $scripts));
             exit();
         }
-        echo $styles, $output, $scripts;
+        echo wp_kses($output . $scripts, AllowedTags::getWithScriptAndStyleTags());
```

Comment:
Removed the styles from here and added them as a new file located in `core/templates/global_assets/css/ee-exception-stack-display.css` path. Added the link tag to the HTML codes. Also modified the `AllowedTags` class to support link tag.

Since this class can be called from both of frontend and backend, I was going to add the CSS codes to a file that is loading in both of them but didn't find anything so I added the new file as mentioned earlier. I'm not sure if adding the file, path of it and naming is correct so if there is anything just let know to correct it.

Reply:
the `core/templates/global_assets/css/ee-exception-stack-display.css ` file name and location is PERFECT 💯